mlflow/gateway/schemas/completions.py

```diff
@@ -26,7 +26,7 @@ class CandidateMetadata(ResponseModel):
 
 class Candidate(ResponseModel):
     text: str
-    metadata: CandidateMetadata
+    metadata: Optional[Dict]
```

Comment:
```suggestion
    metadata: Optional[Dict[..., ...]]
```

Is it possible to set dict key and value types?